test/server.js

```diff
@@ -26,6 +26,7 @@ nunjucks.configure('./test/views/', {noCache: true});
 // Turn off all caching for tests.
 app.use((req, res, next) => {
   res.set('Cache-Control', 'no-cache');
+  res.set('Access-Control-Allow-Origin', '*');
```

Comment:
Unrelated to this PR, but since I was testing my local changes with the INP Demo page, it was useful to be able to load code from the `/dist` folder over localhost (which is generally useful), so I decided to keep this in.